Sources/BraintreePayPalNativeCheckout/BTPayPalNativeRequest.swift

```diff
@@ -105,58 +105,59 @@ import BraintreePayPal
         switch paymentType {
         case .checkout:
             guard let request = request as? BTPayPalNativeCheckoutRequest else { return [:] }
+            var checkoutParameters: [String: Any] = [
+                "intent": request.intentAsString,
+                "amount": request.amount,
+                "offer_pay_later": request.offerPayLater
+            ]
 
-            var billingAgreementDictionary: [AnyHashable: Any]? = [:]
+            let currencyCode = request.currencyCode != nil ? request.currencyCode : configuration.json["paypal"]["currencyIsoCode"].asString()
```

Comment:
```suggestion
            let currencyCode = request.currencyCode ?? configuration.json["paypal"]["currencyIsoCode"].asString()
```

I think we can use the nil coalescing operator here